docs/DeveloperGuide.md

```diff
@@ -70,16 +71,16 @@ Each of the four components
 * Exposes its functionality using a `{Component Name}Manager` class e.g. `LogicManager.java`
 
 The _Sequence Diagram_ below shows how the components interact for the scenario where the user issues the
-command `delete 3`.
+command `delete 1`.
 
-<img src="images\SDforDeletePerson.png" width="800">
+<img src="images\SDforDeleteTask.png" width="800">
 
 >Note how the `Model` simply raises a `ModelChangedEvent` when the model is changed,
  instead of asking the `Storage` to save the updates to the hard disk.
 
 The diagram below shows how the `EventsCenter` reacts to that event, which eventually results in the updates
 being saved to the hard disk and the status bar of the UI being updated to reflect the 'Last Updated' time. <br>
-<img src="images\SDforDeletePersonEventHandling.png" width="800">
+<img src="images\SDforDeleteTaskEventHandling.png" width="800">
```

Comment:
The sequence diagram here has a bit of problem. First of all there should be an activation bar for the "Post()" Event, 2ndly, if your second diagram has double flow then it's not so good, you need to think of a way to improve it. :)